pkgs/applications/networking/modem-manager-gui/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, buildEnv, pkgconfig, python3, fetchhg, gtk3-x11, glib, gdbm, gtkspell3, itstool, libappindicator-gtk3, meson, ninja, perldevelPackages }:
+
+stdenv.mkDerivation rec {
+  name = "modem-manager-gui";
+  version = "0.4.17";
+  rev = "v${version}";
+
+  src = fetchhg {
+    url = https://linuxonly@bitbucket.org/linuxonly/modem-manager-gui;
+    rev = "8bcf842";
+    sha256 = "04p08gzvi068bd3slbnaghajwzp4h5jacib640k5shngqimkl84p";
+  };
+
+  buildInputs = [
+    pkgconfig
+    gtk3-x11
```

Comment:
Why not just `gtk3`?

Reply:
I didn't knew it existed - `nix-env -f "<nixpkgs>" -qaP|grep gtk3` doesn't list gtk3, only gtk3-x11. Is there a command that would have listed gtk3?

Reply:
fixed